Dockerfile.ci

```diff
@@ -0,0 +1,22 @@
+ARG image_tag=develop
+FROM elifesciences/proofreader-php_composer_dev:${image_tag} AS build
+FROM elifesciences/php_7.0_cli:656bb4bdf1e49a5e80337e2a7c4f44f10c3f52b0
```

Comment:
the base image version should probably start to be moved to `.env`? ease of update